example/Validations.res

```diff
@@ -1,35 +1,32 @@
 include Formidable.Validations
-open Validator.Args
-open Optic
 
-let required = (
-  list{"required"},
-  ({label, value}) =>
-    if value == "" {
-      #error(#error(("required", label)))
-    } else {
-      #ok(value)
+let required = {
+  Description.names: list{"required"},
```

Comment:
I went for a record instead of a tuple as they're not slower and more explicit 👍